.env.defaults

```diff
@@ -14,6 +14,18 @@ KEEP_ALIVE_TIMEOUT_MS=
 # functioning correctly.
 CURVEBALL_ORIGIN="http://localhost:8531"
 
+# Allow CORS from any origin by running the seed script 002_cors_allowed_origin.js
+# ie. CORS_ALLOW_ORIGIN=["http://localhost:8531", "http://localhost:3000"]
+CORS_ALLOW_ORIGIN=
+
```

Comment:
I was gonna leave a comment that this shouldn't use seed files and instead use the built-in facility for loading config from .env, but it looks like you partially did this?

This is not the type of thing that should ever exists in a seed file. Remember that seed files are contributor only and just meant for dev data.

Reply:
Seed files should effectively behave as a second set of migration files, ran in order and maintaining consistency over time.

Reply:
The addition of `CORS_ALLOW_ORIGIN` is meant to benefit both end users and contributors by allowing them to have a quick way to manipulate those values without having to directly run database code

Reply:
Yes, 100% agree. This is a useful environment variable. I mentioned I wanted to add this at our last meeting. But the problem is that you implemented it with seed files. Seed files are *not* used by end-users.

Reply:
So what may be the alternative or better practice?

Reply:
>  built-in facility for loading config from .env, but it looks like you partially did this?

Like... `src/env.ts`?

Reply:
Gonna leave CORS_ALLOW_ORIGIN for a differnet PR